proposals/graduation/cert-manager.md

```diff
@@ -0,0 +1,512 @@
+
+- Share steering committee meetings and notes as stated per governance: “All recordings and minutes will be posted publicly”, including  progress, input or impact of the from the committee on the project.
+- Update the Roadmap to reflect the ongoing work and planned efforts. Also, as part of the governance, document the roadmap change process.
+- Provide an up-to-date list of sub-projects to be included as part of the Graduation evaluation.
+- Provide a RELEASE.md file, including updating the release process to reflect the latest shift to the CNCF infrastructure.
+- Update the Code of Conduct to reference the CNCF Code of Conduct. Currently the Code of Conduct is adapted from the Contributor Covenant, however one of the incubation criteria is to explicitly reference and use the CNCF CoC https://github.com/cncf/foundation/blob/main/code-of-conduct.md.
+
+The following recommendations were provided to the project that are non-blocking in the TOC's assessment but should be completed by the project to ensure the continued viability of the project:
+- Complete a joint assessment with TAG Security reviewers.
+- TAG Security encourages the expansion of the current use of go vet and implement govulncheck as planned. It would be beneficial to update your OpenSSF Best Practices information if these have been addressed.
+- TAG Security recommends completing the silver and gold level criteria, as the project likely already meets most of them.
+- TAG Security suggests considering security audits for sub-projects like trust-manager and csi-driver. Perhaps the CNCF can batch these in a follow-up audit.
+- TAG Contributor Strategy recommends to add in/out of scope information to the readme, roadmap, or contributor documentation.
+- TAG Contributor Strategy recommends to add role qualifications for each step on the contributor ladder.  This is particularly important for the maintainers, where other docs refer to Maintainer qualifications that don't exist.
+- TAG Contributor Strategy recommends to add a process for removing Maintainers (and SC members) for reasons other than inactivity, such as violating the CoC or disruptive behavior.
+- TAG Contributor Strategy recommends to link to the list of official channels and meetings in the contributor docs from the Governance document.
+- TAG Contributor Strategy recommends to gradually build up the sub-projects into their own entities, allowing new contributors to take ownership of them.  This will require adding to the main Governance for these roles.
+- TAG Contributor Strategy recommends to have another Steering meeting, in order to keep Steering members engaged.
+- TAG Contributor Strategy recommends to make sure that the Community repo is linked from appropriate other places, like the main development repos and the contributor docs.
+- TAG Contributor Strategy recommends to figure out a low-effort way to record maintainer decisions for posterity, such as a simple text log.
```

Comment:
@kgamanji Do you know if the project has open issues to track these to resolution?  If they do, it'd be lovely to have them here so anyone can view their status if they come across the information from the DD.

Reply:
@SgtCoDFish will look into creating issues for the recommendation above

Reply:
In the short term, I've created a tracking issue for this work: https://github.com/cert-manager/community/issues/35

We'll go through and create subtasks linked to that issue where needed, and some of the work is already done (e.g. `govulncheck` / `go vet` is done already and it's just a matter of updating docs really)!

Reply:
Awesome thanks so much!!